EngineLayer/Psm.cs

```diff
@@ -36,6 +37,23 @@ public Psm(CompactPeptideBase peptide, int notch, double score, int scanIndex, I
             this.ScanPrecursorMonoisotopicPeak = scan.PrecursorMonoisotopicPeak;
             this.ScanPrecursorMass = scan.PrecursorMass;
             AddOrReplace(peptide, score, notch);
+            this.ExcelCompatible = true;
+        }
+
+        public Psm(CompactPeptideBase peptide, int notch, double score, int scanIndex, IScan scan, bool excelCompatible)
```

Comment:
What's the motivation for having this constructor?

Reply:
Constructor is used for search engines to determine if the psmtsv output should crop or not.

Reply:
Use the  " :this()" syntax, to have a single line in the constructor body

Reply:
can you send me the page you found with this? I'm having trouble using it.

Reply:
jk, got it